YDotNet/Document/Types/Branches/Branch.cs

```diff
@@ -36,10 +38,11 @@ internal protected Branch(nint handle, Doc doc, bool isDeleted)
 
                 return (BranchChannel.ObserveDeep(branch, nint.Zero, callback), callback);
             },
-            (branch, s) => BranchChannel.UnobserveDeep(branch, s));
-#pragma warning restore CA1806 // Do not ignore method results
+            SubscriptionChannel.Unobserve);
     }
 
+    internal BranchIdNative? BranchId { get; }
```

Comment:
I just made a quick search, but I think this could be a private field.

Reply:
@SebastianStehle Sure! Fixed by [this commit](https://github.com/y-crdt/ydotnet/commit/9298507a169665c9d86bf03f84fc83ef4abaee80).